services/base.js

```diff
@@ -334,9 +334,15 @@ class BaseService {
       logoColor: overrideLogoColor,
       logoWidth: overrideLogoWidth,
       link: overrideLink,
-      colorA: overrideLabelColor,
-      colorB: overrideColor,
     } = overrides
+    // Scoutcamp converts numeric query params to numbers. Convert them back.
+    let { colorB: overrideColor, colorA: overrideLabelColor } = overrides
+    if (typeof overrideColor === 'number') {
```

Comment:
Would it be better to check if ` !== 'string'`? in case Scoutcamp also sends booleans or other types of data?